scripts/gasCompare.js

```diff
@@ -0,0 +1,80 @@
+const fs = require("fs");
+const markdown = require('json-to-markdown-table');
+const Commenter = require('../node_modules/circleci-pr-commenter/dist/index');
+const commenter = new Commenter()
+
+let arguments = process.argv
+
+let getFileData = (filePath) => {
+
+    try{
+        const fileData = fs.readFileSync(filePath);
+        const response = JSON.parse(fileData)
+        return response.info.methods;
+
+    } catch (err) {
+        console.log(err);
+        return
+    }  
+
+}
+//method to calculate median value
+
+let calculateMedianValue = (numbers) => {
+    const sorted = numbers.slice().sort((a, b) => a - b);
+    const middle = Math.floor(sorted.length / 2);
+
+    if (sorted.length % 2 === 0) {
+        return (sorted[middle - 1] + sorted[middle]) / 2;
+    }
+
+    return sorted[middle];
+
+}
+//method to compare gas values before and after.
+
+let compareValue = (a,b) => {
+
+    if(a<b){
+        return ((a-b)*100)/a;
+    }
+    else if(b<a){
+        return ((a-b)*100)/a ;
+    }
+    else{
+        return 0
```

Comment:
This function can be optimised for readability.

Reply:
ok